app/helpers/admin/claims_helper.rb

```diff
@@ -1,5 +1,6 @@
 module Admin
   module ClaimsHelper
+    include Admin::PresenterMethods
```

Comment:
Are these presenter methods still needed here?

Reply:
Good catch, this was a leftover from earlier work. I've removed it